src/style.css

```diff
@@ -0,0 +1,145 @@
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.body {
+  display: flex;
+  justify-content: center;
+  color: #060b35;
+  margin: 0;
+}
+
+.body__body {
+  display: flex;
+  flex-direction: column;
+  width: 198px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.body__item-picture {
+  display: flex;
+  padding: 32px 19px 0;
```

Comment:
do not set paddings for each element inside block separately

set padding for the whole card once instead

Reply:
and use mostly margin bottom for vertical indents between elements inside card